### ISSUE TITLE
maxlength in passwordrules not working 

### ISSUE THREAD

**Issue by Pushpak-R** [apple/password-manager-resources#485]
Safari suggesting passwords with random length even when maxlength is specified if there are 2 password fields (password and confirm password). I am using rule like ` passwordrules="minlength: 1; maxlength: 5; required: digit;"` in both fields but Safari is suggesting 596688170883728 password.

**Comment by rmondello** [comment]
This sounds like it might be a Safari bug, not an issue appropriate for this open source project. @Pushpak-R, would you be willing to send feedback to Apple about this?

**Comment by Pushpak-R** [comment]
@rmondello , yes I want to send feedback to Apple regarding this, but I do not know the portal where I should post this.

**Comment by kjhx** [comment]
Perhaps [Feedback Assistant](https://feedbackassistant.apple.com)?
